<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProfileCreate</class>
 <widget class="QDialog" name="NewProfileCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Profile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_11">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Profile Name:</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Copy Profile Name:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_copyProfileName</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_copyProfileName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Source</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QTextEdit" name="textEdit_Source"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>Target</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QTextEdit" name="textEdit_Target"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="Properties">
              <attribute name="title">
               <string>Properties</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="title">
                  <string>Profile Name</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="1" column="0">
                   <widget class="QGroupBox" name="groupBox_6">
                    <property name="title">
                     <string>Description</string>
                    </property>
                    <widget class="QTextEdit" name="textEdit_description">
                     <property name="geometry">
                      <rect>
                       <x>10</x>
                       <y>34</y>
                       <width>201</width>
                       <height>101</height>
                      </rect>
                     </property>
                    </widget>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLineEdit" name="lineEdit_ProfileName">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QGroupBox" name="groupBox_9">
                    <property name="title">
                     <string>Logging</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_9">
                     <item row="0" column="0" colspan="2">
                      <layout class="QHBoxLayout" name="horizontalLayout_8">
                       <item>
                        <widget class="QLabel" name="label_2">
                         <property name="text">
                          <string>Level:</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QComboBox" name="comboBox_loggingLevel"/>
                       </item>
                      </layout>
                     </item>
                     <item row="1" column="0">
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>130</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item row="1" column="1">
                      <widget class="QPushButton" name="pushButton_LogFilebrowse">
                       <property name="text">
                        <string>Browse</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0" colspan="2">
                      <layout class="QVBoxLayout" name="verticalLayout_16">
                       <item>
                        <widget class="QLabel" name="label_3">
                         <property name="text">
                          <string>FileName</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLineEdit" name="lineEdit_LogFileName"/>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_14">
                   <item>
                    <widget class="QCheckBox" name="checkBox_ProfileEnabled">
                     <property name="text">
                      <string>Enabled</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="checkBox_AutoCommitEnabled">
                     <property name="text">
                      <string>AutoCommit</string>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="checkBox_ProfileTraceEnabled">
                     <property name="text">
                      <string>Trace</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="checkBox_ProfileTransactionLogging">
                     <property name="text">
                      <string>Logging</string>
                     </property>
                     <property name="checked">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_4">
                   <property name="title">
                    <string>Commit Throttle</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                     <widget class="QSpinBox" name="spinBox_CommitThrottle">
                      <property name="maximum">
                       <number>100000</number>
                      </property>
                      <property name="value">
                       <number>1000</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="title">
                    <string>Execution Threads</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_5">
                    <item row="0" column="0">
                     <widget class="QSpinBox" name="spinBox_ExecutionThreads">
                      <property name="value">
                       <number>1</number>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <spacer name="verticalSpacer">
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>40</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                    <item row="2" column="0">
                     <widget class="QGroupBox" name="groupBox_8">
                      <property name="title">
                       <string>Drop Target</string>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_8">
                       <item row="0" column="0">
                        <widget class="QComboBox" name="comboBox_dropTarget"/>
                       </item>
                      </layout>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QGroupBox" name="groupBox_10">
                      <property name="title">
                       <string>Packet Size</string>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_12">
                       <item row="0" column="0">
                        <widget class="QSpinBox" name="spinBox_NetworkPacketSize">
                         <property name="maximum">
                          <number>32767</number>
                         </property>
                         <property name="value">
                          <number>2048</number>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabTables">
              <attribute name="title">
               <string>Tables</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QListWidget" name="listWidget"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QPushButton" name="pushButton_profileCheck">
       <property name="text">
        <string>Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Generate">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Commit">
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_editProfile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deleteProfile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_copyProfile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="title">
            <string>Action</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QDialogButtonBox" name="buttonBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="standardButtons">
               <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>SQL Statements</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="tab_create">
           <attribute name="title">
            <string>Create Statement</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QListWidget" name="listWidget_createStatements"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_insert">
           <attribute name="title">
            <string>Insert Statement</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidget_insertStatements"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_update">
           <attribute name="title">
            <string>Update Statement</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidget_updateStatements"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_SqlStatement">
           <attribute name="title">
            <string>Sql Statement</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidget_SqlStatements"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTextEdit" name="textEdit_messages"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>NewProfileCreate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>NewProfileCreate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
